src/researchhub/wsgi.py

```diff
@@ -6,17 +6,14 @@
 For more information on this file, see
 https://docs.djangoproject.com/en/2.2/howto/deployment/wsgi/
 """
-import newrelic.agent
+
 import os
 
 from django.core.wsgi import get_wsgi_application
 
-CELERY_WORKER = os.environ.get('CELERY_WORKER', False)
-APP_ENV = os.environ.get('APP_ENV') or 'development'
-
-if not CELERY_WORKER and 'development' not in APP_ENV:
-    newrelic.agent.initialize('./newrelic.ini', APP_ENV)
+CELERY_WORKER = os.environ.get("CELERY_WORKER", False)
```

Comment:
Seems like `CELERY_WORKER` is used in a couple of places like `wsgi.py`, `settings.py` and `asgi.py`. I'm not sure whether it is needed for celery related purpose

Reply:
I am not removing this, it just got reformatter (isolated in commit 556e3549a2ff17e85166b0e04c05cf68e183c926).

Reply:
Ah oops.